README.md

```diff
@@ -1,13 +1,22 @@
-# Project Name
+# Chatbot
 
 Replace this readme with your own information about your project.
 
 Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
 
```

Comment:
It would have been helpful with just a short description of the assignment here.